### PR TITLE
Initial split of repo for IT

### PR DESCRIPTION
**Pull request by iderdik**
Separates etcd bootstrapping into separate service and container for IT cloud.

### CODE REVIEW

**Comment by nBerg** [comment]
:+1:

File: init [hunk 1]
```diff
@@ -15,6 +15,10 @@ if [ "$NODE_ROLE" = "control" ]; then
   sudo bash ${SCRIPTDIR}/v2/util/etcd2-setup.sh $SCRIPTDIR
 fi
 
+if [ "$NODE_ROLE" = "it-hybrid" ]; then
+  sudo bash ${SCRIPTDIR}/itv1/util/etcd2-setup.sh $SCRIPTDIR
```
Comment (chr0n1x) [inline init:19]: lol

**Comment by chr0n1x** [comment]
:+1:

**Comment by matthewdfuller** [comment]
:+1:

**Pull request merged**